src/Toolbar/Toolbar.react.js

```diff
@@ -296,6 +296,12 @@ class Toolbar extends PureComponent {
             return null;
         }
 
+        if (!this.state.isSearchActive && React.isValidElement(leftElement)) {
+            return (
+              React.cloneElement(leftElement, { key: 'customLeftElement' })
```

Comment:
First of all, thank you @pfulop. I think, we should wrap it into the `styles.leftElementContainer`, since the right element is wrapped in `styles.rightElemenentContainer` as well. What do you think?